proto/lib/python/slice.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package syft.lib.python;
+import "proto/core/common/common_object.proto";
+
+message Slice {
+  repeated bytes start = 1;
```

Comment:
Can't these just be normal `ints`?

Reply:
yes, that won't be a problem but i just thought if it might help with faster computation

Reply:
The problem would be that we need to convert the ints into syft.lib.python.Ints then to protos and bytes, where as i'm sure the built in protobuf integer type is extremely fast and lightweight.

Reply:
Alright!